sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -71,10 +77,23 @@ internal class ViewAnnotationManagerImpl(
     val annotation = annotationMap.remove(id) ?: return false
     annotation.view.removeOnAttachStateChangeListener(annotation.attachStateListener)
     mapView.removeView(view)
+    notifyViewAnnotationVisibilityChanged(annotation, false)
     getValue(mapboxMap.removeViewAnnotation(id))
     return true
   }
 
+  override fun removeAllViewAnnotations() {
+    annotationMap.forEach { (id, annotation) ->
+      getValue(mapboxMap.removeViewAnnotation(id))
+      annotation.view.removeOnAttachStateChangeListener(annotation.attachStateListener)
+      annotation.attachStateListener = null
```

Comment:
Compared with `removeViewAnnotation` why have `annotation.attachStateListener = null` here and no     `notifyViewAnnotationVisibilityChanged(annotation, false)`?